doc/api/dgram.md

```diff
@@ -95,6 +95,17 @@ Tells the kernel to join a multicast group at the given `multicastAddress` and
 one interface and will add membership to it. To add membership to every
 available interface, call `addMembership` multiple times, once per interface.
 
+When the socket is created, each worker gets a copy and they refer the
```

Comment:
Does "worker" refer to a cluster environment? If so I think it makes sense to specify it as the sentence seems to be a bit out of context without it.

Reply:
Yes, it does and I agreed on it. I will add the context.